integration_tests/recorder-extension/src/agent_payload.proto

```diff
@@ -0,0 +1,185 @@
+syntax = "proto3";
+
+// file copied from https://github.com/DataDog/agent-payload/blob/master/proto/metrics/agent_payload.proto
+
```

Comment:
I think I may have left a comment to this effect before but I still think we should remove the code in this file that is not used by our tests. I know that you copied this file from elsewhere, but to me it still seems unnecessary to have some unrelated, unused Kubernetes-related code in our integration tests. I think it would be better to just copy the code that we use in these tests.

Reply:
✅